internal/backend/sessions/sessionworkflows/config.go

```diff
@@ -18,4 +18,6 @@ type Config struct {
 	OSModule bool `koanf:"os_module"`
 
 	SlowOperationTimeout time.Duration `koanf:"slow_operation_timeout"`
+
+	Test bool `koanf:"test"`
```

Comment:
Please add a comment that explains the purpose/function of this flag.

Reply:
done